website/docs/components/rich-tooltip/partials/code/component-api.md

```diff
@@ -0,0 +1,125 @@
+  </C.Property>
+  <C.Property @name="[RT].isOpen" @type="yielded tracked property">
+    Hook into this tracked property to access the state of `isOpen`.
+  </C.Property>
+  <C.Property @name="[RT].close" @type="yielded function">
+    Hook into this function to programmatically close the toggle.
+  </C.Property>
+  <C.Property @name="...attributes">
+    This component supports use of [`...attributes`](https://guides.emberjs.com/release/in-depth-topics/patterns-for-components/#toc_attribute-ordering).
+  </C.Property>
+</Doc::ComponentApi>
+
+### Contextual components
+
+The Toggle and Bubble elements are passed into the Rich Tooltip as yielded components, using the `Toggle` and `Bubble` keys.
+
+#### RichTooltip::Toggle
+
+Standard toggle element to use, made of a text and an optional icon, to ensure that the toggle is perceivable, visually consistent, and can be used inline with other content or standalone as part of the layout flow.
+
+It can also be used with generic content, in which case consumers will need to make sure the component is used in a [conformant accessible way](/components/rich-tooltip?tab=accessibility).
+
+<Doc::ComponentApi as |C|>
+  <C.Property @name="text" @type="string">
+    Plain text string that will appear as `<span>` HTML element inside the toggle button. It has a dotted underline styling applied to it.
+  </C.Property>
+  <C.Property @name="icon" @type="string">
+    Use this parameter to show an icon. Any [icon](/icons/library) name is acceptable. The icon size is proportional to the font size (`1em`).
+    <br />
+    <br />
+    _If the toggle contains only the icon, an `aria-label` value needs to be assigned to the toggle to make it accessible._
+  </C.Property>
+  <C.Property @name="iconPosition" @type="enum" @values={{array "leading" "trailing"}} @default="trailing">
+    Positions the icon before or after the text.
+  </C.Property>
+  <C.Property @name="size" @type="string" @values={{array "undefined" "small" "medium" "large"}} @default="undefined">
+    If undefined (default), the `@text` inherits its typographic styles from the parent element.
+    <br />
+    If instead the `@size` is defined, a typographic style (and font size) is applied  to the `@text` according to the scale `body-100/200/300`.
+    <br />
+    <br />
+    _⚠️ Important: the size is applied only to the `@text`, not the `yield` content._
+  </C.Property>
+  <C.Property @name="isInline" @type="boolean" @default="false" @values={{array "true" "false"}}>
+    Sets the display for the `<button>` HTML element of the "toggle" content. If set to `true` it sets the element's display to `inline-flex` (a `<button>` can't have an `inline` layout).
+  </C.Property>
+  <C.Property @name="yield" @type="yield">
+    It's possible to yield generic content to the "toggle" element, instead of using the `@text` argument. This should be used only in special/edge cases where the toggle based on text/icon doesn't work for the specific context or design.
```

Comment:
```suggestion
    It’s possible to yield generic content to the "toggle" element, instead of using the `@text` argument. This should be used only in special edge cases where including text or an icon in the toggle doesn’t work for the specific context or design.
```